devimint/src/vars.rs

```diff
@@ -93,6 +93,11 @@ declare_vars! {
         FM_ESPLORA_DIR: PathBuf = mkdir(FM_TEST_DIR.join("esplora")).await?;
         FM_READY_FILE: PathBuf = FM_TEST_DIR.join("ready");
 
+        // variables required by the admin client
```

Comment:
There's bunch of a things in this PR that seem unrelated (new env vars, passing `WsAdminClient`). Is this intentional?

Reply:
We shouldn't hard-code `FM_OUR_ID`

Reply:
I was trying the hard way, which is to add to cli ng, but let's try now the easy way, which is much simpler